doc/TechnicalFAQs.md

```diff
@@ -0,0 +1,128 @@
+ # Technical Troubleshooting Guide
+
+This guide serves as a collection of technical issues/solutions commonly seen when trying to install PowerPointLabs.
+
+If better solutions to any issue are found, or if there is a need for any clarification, please kindly let us know at pptlabs@comp.nus.edu.sg
+
+### Table of Issues
+1. [Certificate Not Trusted](#cert-not-trusted)
+1. [VSTO Installer Config](#vsto-installer-config)
+1. [NET Framework Not Installed](#net-not-installed)
+1. [VSTO Opens as XML](#vsto-as-xml)
+1. [Missing PowerPointLabs Ribbon](#missing-ribbon)
+1. [Renaming VSTOInstaller.exe.config File](#rename-vsto-file)
+1. [ClickOnce Cache](#clickonce-cache)
+1. [Referenced Assembly Not Installed](#assembly-not-installed)
+1. [Corrupted Zip Installer](#corrupted-zip)
+1. [Preexisting Similar Application](#preexisting-application)
+1. [Error Loading Manifest from MahApps.Metro.dll](#error-manifest)
```

Comment:
Phrase all of them as problems.
* `Certificate Not Trusted` ✓
* `ClickOnce Cache` :x: (not phrased as a problem)